### ISSUE TITLE
Single Use Token Not Installing

### ISSUE THREAD

**Issue by michaelalhilly** [envato/wp-envato-market#64]
Hi. When I attempt to use the single use token to install a theme I am receiving the following error:

Install package not available.

I am able to install the theme with the global personal token. I am also able to update the theme with both the global and single use tokens.

I am using the latest version of the plugin on the latest version of WP. My stack is Debian 8, Nginx 1.11, PHP-FPM 7, and MariaDB 10.

Here are my token settings:
- View and search Envato sites
- Download your purchased items
- List purchases you've made
- Verify purchases you've made

Let me know if I should provide more info.

**Comment by ciberdany** [comment]
i have the same error.

wp and plugin last version and i don't know how to fix, i don't want that my client see all my envato purchases.

**Comment by michaelalhilly** [comment]
@ciberdany After some testing I found that this error was related to the theme I purchased as this feature is working with other themes from Themeforest. I would suggest contacting the theme developer.

**Comment by GabrielAPass** [comment]
Can either of you tell me if you found any information on what the problem was that caused the error.

I am having the same problem, but the developer isn't sure why.
Thanks.

**Comment by drugurkocak** [comment]
Hi,
I purchased Envision theme for Wordpress and tried to install using global and single use tokens.
I remember I succeeded once using Single Use OAuth Personal Tokens. Today, I can see the theme name under the Themes tab, but I get the same error "package not available" when I try to install via Envato Market plugin. 
[Envision](https://themeforest.net/item/envision-responsive-retina-multipurpose-theme/5928018)
